### PR TITLE
Add logs for UniswapV3 pool fetching

### PR DESCRIPTION
**Pull request by sunce86**
Added logs for UniswapV3 pool fetching and caching. I want to make sure the caching is working as expected and check if any tweaks are needed.

Expected to remove the logs after the analysis.

### CODE REVIEW

**Review by ahhda — APPROVED** [review]
LGTM

File: crates/shared/src/sources/uniswap_v3/pool_fetching.rs [hunk 1]
```diff
@@ -132,7 +132,9 @@ impl UniswapV3PoolFetcher {
     }
 
     async fn get_pools_and_update_cache(&self, pool_ids: &[H160]) -> Result<Vec<PoolData>> {
+        tracing::debug!("get_pools_and_update_cache");
         let pools = self.graph_api.get_pools_with_ticks_by_ids(pool_ids).await?;
+        tracing::debug!("pools len: {}", pools.len());
```
Comment (bh2smith) [inline crates/shared/src/sources/uniswap_v3/pool_fetching.rs:137]: Nit: I feel like you could use the full word length here.

File: crates/shared/src/sources/uniswap_v3/pool_fetching.rs [hunk 2]
```diff
@@ -229,6 +245,7 @@ async fn update_recently_used_outdated_pools(
     update_size: Option<usize>,
 ) {
     while let Some(inner) = inner.upgrade() {
+        tracing::debug!("periodical update started");
```
Comment (bh2smith) [inline crates/shared/src/sources/uniswap_v3/pool_fetching.rs:248]: Nit: periodical is a term used for periodically publishing articles, might be better to use just "periodic"

cf - https://grammarist.com/usage/periodic-periodical/

But please feel free to ignore.

File: crates/shared/src/sources/uniswap_v3/pool_fetching.rs [hunk 3]
```diff
@@ -256,6 +275,7 @@ async fn update_recently_used_outdated_pools(
             }
         }
 
+        tracing::debug!("periodical update ended");
```
Comment (bh2smith) [inline crates/shared/src/sources/uniswap_v3/pool_fetching.rs:278]: Ditto - periodic

**Review by bh2smith — COMMENTED** [review]
Oh, I guess I'm too late but this was "approve" with grammatical nits.

**Comment by sunce86** [comment]
> Oh, I guess I'm too late but this was "approve" with grammatical nits.

Thanks Ben. Usually, I would make a following PR with the fixes but, in this case, since this PR is scheduled to be reverted once the analysis is over, I believe there is not point.

**Pull request merged**